pyintelowl/cli/jobs.py

```diff
@@ -84,6 +85,19 @@ def view(ctx: ClickContext, job_id: int, categorize: bool, as_json: bool):
         ctx.obj.logger.fatal(str(e))
 
 
+@jobs.command(help="Kill a running job by job ID")
+@click.argument("job_id", type=int)
+@click.pass_context
+def kill(ctx: ClickContext, job_id: int):
+    ctx.obj.logger.info(f"Requesting kill for Job [underline blue]#{job_id}[/]..")
+    try:
+        ans = ctx.obj.kill_running_job(job_id)
+        if ans:
+            rprint("killed")
```

Comment:
Should probably be emphasized with some underline/bold and red colored.

Reply:
And we should display 'not killed' too, in case `ans` is false.

Reply:
So when it successfully gets killed, render it with green color as a representation of success and when it doesn't get killed render it with red color.

Reply:
Actually `ans` is true only when the `STATUS 200`. `IntelOwlClientException` would be raised in the other case, so it won't reach here. A refactor would be needed

Reply:
I just saw I pushed the commit without that exception being raised. If only need a bool state, this is fine then.

Reply:
I can return the exception from `kill_running_job` and raise it once I display the text.

Reply:
![image](https://user-images.githubusercontent.com/51958314/107987024-488f0280-6ff3-11eb-8a2a-020e0cdb86f0.png)

Reply:
This looks good I guess.

Reply:
> I can return the exception from `kill_running_job` and raise it once I display the text.

No, can't do that. This will be hard to understand/handle for the users using pyintelowl as a library. You can just `rprint("Failed to kill")` in the except block before the log.error statement.

Reply:
Hmm, my bad.